core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -80,6 +80,8 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
     /(<(oppia-noninteractive-(.+?))\b[^>]*>)[\s\S]*?<\/\2>/g
   );
 
+  @ViewChild('oppiaRTE') oppiaRTE: ElementRef;
```

Comment:
This property is unused, Why was this added? @Lawful2002

Reply:
This code is not used, it was meant for testing purposes. It was left in by mistake.